frontend/src/tests/components/commons/Tab.test.js

```diff
@@ -6,13 +6,8 @@ it("renders selected tab", () => {
   const wrapper = shallow(<Tab tabName="Tab1" selected={true} />);
   const initialMessage = (
     <span>
-      <li role="presentation" style={styles.li} aria-current="page">
-        <button
-          id="Tab1"
-          style={{ ...styles.button, ...styles.active }}
-          role="tab"
-          aria-selected="true"
-        >
+      <li role="menuitem" style={styles.li} aria-current="page">
+        <button id="Tab1" style={{ ...styles.button, ...styles.active }} aria-selected="true">
```

Comment:
I know this isn't part of your code, but do we actually need these id's? If so, can they use lowecase style?

Reply:
Will arial-selected dynamically change based on what's actually selected?

Reply:
I will look into the ids; the tab has a prop (selected=true/false) that is passed in. aria-selected and aria-current are defined based on that

Reply:
dropped ids and fixed related tests in e525b99